tests/monotouch-test/NearbyInteraction/NINearbyObjectTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright 2020 Microsoft Inc.
+
+#if !MONOMAC
+
+using System;
+using Foundation;
+using ObjCRuntime;
+using NearbyInteraction;
+using NUnit.Framework;
+using OpenTK;
+
+namespace MonoTouchFixtures.NearbyInteraction {
+
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class NINearbyObjectTest {
+
+		[Test]
+		public void DirectionNotAvailable ()
+		{
+			if (!TestRuntime.CheckXcodeVersion (12, 0))
+				Assert.Ignore ("Requires iOS 14.0");
+
+			Vector3 vect = NINearbyObject.DirectionNotAvailable;
+
+			unsafe {
+				Vector3* v = &vect;
+				byte* ptr = (byte*) v;
+				byte zero = 0;
+				for (var i = 0; i < sizeof (Vector3); i++)
+					Assert.True (ptr[i] == zero, "Expected {0} but was {1}", zero.ToString(), ptr[i].ToString());
```

Comment:
minor:
* style, missing space before `(` (twice)
* `zero.ToString()` allocates and it's a constant in the loop
* the assert error will tell you a byte is not `0` but won't tell you where it is (`I`)

```csharp
Assert.That (ptr [i], Is.EqualTo (zero), $"Position {i}");
```

The above assert will provide the expected and actual values (and turn them to string only if needed/failing) so the only missing part (to diagnose the failure) is the position `i`.